02-peer-protocol.md

```diff
@@ -644,6 +645,118 @@ transactions may be out of sync indefinitely. This is not concerning:
 what matters is whether both sides have irrevocably committed to a
 particular update or not (the final state, above).
 
+### Simplified Operation
+
+If `option_simplified_update` is negotiated, a subset of the previous
+protocol is used, where each side takes turns to propose updates,
+which are synchronized before the other side has a turn:
+
+        +-------+                               +-------+
+        |       |--(1)---- update_add_htlc ---->|       |
+        |       |--(2)---- update_add_htlc ---->|       |
+        |       |--(3)--- commitment_signed --->|       |
+        |   A   |<-(4)---- revoke_and_ack ------|   B   |
+        |       |<-(5)--- commitment_signed ----|       |
+        |       |--(6)---- revoke_and_ack ----->|       |
+        |       |                               |       |
+        |       |<-(7)---- update_add_htlc -----|       |
+        |       |<-(8)--- commitment_signed ----|       |
+        |       |--(9)---- revoke_and_ack ----->|       |
+        |       |--(10)-- commitment_signed --->|       |
+        |       |<-(11)--- revoke_and_ack ------|       |
+        +-------+                               +-------+
+
+If one side sends an update when it is not its turn, the other side
+can either ignore it (in favor of its own update), or reply with a
+`yield` message to hand the turn over.
+
+#### Swapping turns: `yield` and `update_noop`
+
+1. type: 129 (`update_noop`) (`option_simplified_update`)
+2. data:
+   * [`channel_id`:`channel_id`]
+
+1. type: 138 (`yield`) (`option_simplified_update`)
+2. data:
+   * [`channel_id`:`channel_id`]
+
+#### Requirements
+
+A node:
+  - MUST NOT send `yield` unless `option_simplified_update` is negotiated.
+  - MUST track whose turn it is, starting with the peer with the lesser SEC1-encoded node_id.
+  - MUST give up its turn when:
+    - sending `revoke_and_ack`
+    - sending a `yield`
+  - MUST accept its turn when:
+    - receiving `revoke_and_ack`
+    - receiving a `yield`
+  - During this node's turn:
+    - if it receives an update message or `commitment_signed`:
+      - if it has sent its own update or `commitment_signed`:
+        - MUST ignore the message
```

Comment:
the remote node sends a commitment_signed during the local node's turn, after the local node has sent a commitment_signed message though? diagram, step (5)...